client/Lykke.Service.ChainalysisProxy.Client/AutofacExtension.cs

```diff
@@ -22,16 +24,34 @@ public static void RegisterChainalysisProxyClient(this ContainerBuilder builder,
                 .SingleInstance();
         }
 
+        public static void RegisterChainalysisProxyClient(this ContainerBuilder builder, string serviceUrl, ILogFactory logFactory, int timeout)
```

Comment:
According to the https://github.com/LykkeCity/Lykke.Logs/blob/master/migration-to-v5.md#client-nugets ILogFactory instance should be resolved inside this method